packages/auth/src/platform_browser/mfa/assertions/phone.test.ts

```diff
@@ -78,7 +80,9 @@ describe('platform_browser/mfa/phone', () => {
           sessionInfo: 'verification-id'
         }
       });
-      expect(session.auth).to.eql(auth);
+      if (session.user) {
```

Comment:
same as a previous comment above: remove this if check and expect session.user to equal "user" and initialize "user".  applies to the instances below too.